pkg/stream/xprotocol/connpool_binding.go

```diff
@@ -57,6 +59,22 @@ func (p *poolBinding) CheckAndInit(ctx context.Context) bool {
 	return true
 }
 
+type AddConnListenerFunc func(downstreamConn api.Connection, binding *activeClientBinding)
+
+func (p *poolBinding) addConnListenerOnce(downstreamConn api.Connection, binding *activeClientBinding) {
```

Comment:
We don't delete `activeClient` from the map when activeClient has closed, and It will cause mem leak. 
maybe we can add a sync.Once in `activeClient`.

Reply:
it looks like that activeClientBinding would be removed when it was closed?